web-components/src/components/taskitem/scss/task-item.scss

```diff
@@ -157,6 +157,7 @@
     display: inline-block;
     font-size: $taskitem-newchat-font-size;
     line-height: $taskitem-base-height;
+    margin-top: 8.5px;
```

Comment:
this has serious consequences. please apply this style in product and not here. please remove

Reply:
Removed